lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -24,17 +58,21 @@ class CircleImageBase extends NodeBase {
   }
 
   /**
-   * Switch between the base and the selected image.
+   * Set selection and switch between the base and the selected image.
+   *
+   * Do the switch only if imageObjAlt exists.
+   *
+   * @param {true|false} selected value of new selected state for current node
    */
   switchImages(selected) {
```

Comment:
Maybe this should now be changed to choseImage or determineImage as its called all the time?

Or, as the this.selected is set here perhaps  handleImageForSelectStatus